src/resources/HostedInterfacesCore/HostedInterfaceCoreConfig.model.ts

```diff
@@ -126,6 +126,13 @@ export interface ListHostedInterfacesParams extends Paginated {
     perPage?: number;
 }
 
+export interface ListHostedInterfacesByProjectParams extends ListHostedInterfacesParams {
```

Comment:
What's the plan with the new interfaces you added? I don't see them used anywhere.

Reply:
They are not used yet, but I went ahead and added them preemptively because eventually we will want to add a projects column for all resource panels supported by the project feature

Reply:
They are going to be used in the admin UI : https://github.com/coveo-platform/admin-ui/pull/12536/files

Reply:
If those are meant to be used solely in the admin UI, wouldn't it make more sense to add them in the admin directly?

Reply:
The reason I added them here is simply because the original param model was in the platform client as well. Did not want to make the call to move everything out or to do things differently. Eventually the client implementation could be moved here as well.

Reply:
> Did not want to make the call to move everything out

Why would declaring those interfaces in the admin require the need to move everything out?

Reply:
My line of thinking was that all the the param and result models for the same resource should be together. Since the original models were in the platform-client, i added mine there. If I wanted it to live in the admin-ui, it would be more coherent to move all models for the search page client to the admin ui. Otherwise we will have some models in the admin-ui and others in the platform-client. Same for IPX.

Reply:
> it would be more coherent to move all models for the search page client to the admin ui

I think that's where we seem to disagree. If types that are solely used in the admin-ui were added in platform-client previously, I don't feel it's a reason to perpetuate this pattern just for the sake of having everything in the same place. It's misleading and it adds unnecessary maintenance overhead. We also have to keep in mind that this package is a public package used not only by the admin-ui but also by our clients. I think we should aim at adding new stuff in the proper place. After that, it makes for less tech debt we need to address.

Now, I really don't mean to be a pain and hinder your work for a smaller issue like this. If you favour your approach, I guess you can go ahead.

Reply:
No no, that's fine ! I was not against your idea, I was just explaining why i did this to begin with. I will change it.

Reply:
done !